cvise/passes/lines.py

```diff
@@ -19,31 +19,39 @@ def __format(self, test_case, check_sanity):
         with (
             CloseableTemporaryFile(mode='w+', dir=tmp) as backup,
             CloseableTemporaryFile(mode='w+', dir=tmp) as tmp_file,
+            CloseableTemporaryFile(mode='w+', dir=tmp) as stripped_tmp_file,
         ):
             backup.close()
             with open(test_case) as in_file:
                 try:
                     cmd = [self.external_programs['topformflat'], self.arg]
-                    proc = subprocess.run(cmd, stdin=in_file, capture_output=True, text=True)
+                    with subprocess.Popen(cmd, stdin=in_file, stdout=subprocess.PIPE, text=True) as proc:
```

Comment:
I took the liberty to change this to read stdout per line, instead of accumulating everything in the memory. It's not a big deal, but the optimization felt easily doable...